### ISSUE TITLE
Changing zoom and position of Camera2D with parallel Tween cause jitter on first and last frame 

### ISSUE THREAD

**Issue by indubitablement2** [godotengine/godot#73298]
### Godot version

4.0 rc2

### System information

Linux - OpenGL API 4.6 (Core Profile) Mesa 22.3.4 - Compatibility - Using Device: Intel - Mesa Intel(R) HD Graphics 5500 (BDW GT2)

### Issue description

When a tween modify the zoom and position of a camera in parallel, there is a small jitter (first frame move positon by a few pixels, last frame will do the opposite movement). 

When doing the same interpolation using lerp in _process or a using single tween_method, that does not happen.

I have tested moving and scaling a sprite, but there is no noticeable jitter.

### Steps to reproduce

Translate and zoom a camera in parallel with a Tween using tween_property and setting parallel to true.

- In `_tween_property_parallel` is the code to move using two tween_property in parallel. It is activated with **up arrow**. There is a small jitter noticeable on the last interpolation frame.
- In `_process` is the code to move using `lerp`. It is activated with **left arrow**. This work fine.
- In `_tween_method` and `_tween_method_callback` is the code to move using one tween_method. It is activated with **right arrow**. This work fine.
- In `_tween_sprite` is the code to move a sprite using two tween_property in parallel. It is activated with **down arrow**. This work fine.

### Minimal reproduction project

[camera bug.zip](https://github.com/godotengine/godot/files/10735087/camera.bug.zip)

**Comment by indubitablement2** [comment]
https://user-images.githubusercontent.com/62550921/220125632-361dce52-d57a-478a-bf94-a14978b37ead.mp4

**Comment by kleonc** [comment]
The cause is different order of applying interpolated values, there's no issue with the Tween parallel interpolation in here.

In the MRP both when interpolating in `_process` and with `Tween.tween_method` the camera's `postion` is set before `zoom`:
```gdscript
func _process(delta: float) -> void:
	(...)
		position = START_POS.lerp(END_POS, weight)
		zoom = START_ZOOM.lerp(END_ZOOM, weight)

func _tween_method_callback(weight: float) -> void:
	position = START_POS.lerp(END_POS, weight)
	zoom = START_ZOOM.lerp(END_ZOOM, weight)
```
But when interpolating with parellel tween the order is swapped (`zoom` before `position`):
```gdscript
func _tween_property_parallel() -> void:
	_prepare_tween()
	_t.tween_property(self, ^"zoom", END_ZOOM, DURATION)
	_t.tween_property(self, ^"position", END_POS, DURATION)
```

In v4.2.1.stable.official [b09f793f5] in all interpolation cases presented above the issue occurs if Camera2D's `zoom` is set before its `position`, and there's no issue if `position` is set before `zoom`.

The issue with the Camera2D causing this is already fixed by #84465 though, which is included since `4.3.dev3` (in there it works the same regardless of the order of `zoom`/`position` assignment).

Closing this as already fixed.
